crates/providers/messaging-nats/wit/deps/messaging/messaging.wit

```diff
@@ -1,50 +1,34 @@
+// Message broker interface
+// This is a phase 1 interface, and is subject to change
+// This interface is used to send and receive messages from a message broker
+
+// Note that in this phase 1 interface, subscriptions are defined out of band from this
+// component. Components themselves cannot establish or terminate subscriptions. This may
+// change for phase 2.
 package wasmcloud:messaging;
 
-/// This interface represents the functions necessary
-/// to implement the [messaging wasmCloud interface](https://github.com/wasmCloud/interfaces/blob/main/messaging/messaging.smithy)
-///
-/// Derived from https://github.com/wasmCloud/interfaces/blob/f020c93d4cacd50318301f686e2f059a15862e1e/messaging/messaging.smithy
-interface messaging {
-    /// A message to be published
-    record message {
-        /// Subject (or topic) of the message
+// Types common to message broker interactions
+interface types {
+    // A message sent to or received from a broker
+    record broker-message {
         subject: string,
-
-        /// Topic on which the reply should be sent
+        body: option<list<u8>>,
```

Comment:
Why did we change this to an option? It seems much cleaner to just have an empty body rather than an optional one, especially since it is a list

Reply:
That's what was implemented in the host for the last ~year. I can change it to a `list<u8>`, but the intention of this PR was just getting to use the same contract in the providers as the host

Reply:
https://github.com/wasmCloud/messaging/pull/4/commits/09f3b13faacb37f05ca31dc061cc1dd22a4a2821

will integrate this now